.github/workflows/nightly-release.yml

```diff
@@ -81,6 +82,7 @@ jobs:
       rom-version: "1.0"
       rom-logging: false
       fpga-itrng: false
+      fpga-runs-on: '["caliptra-fpga-nightly"]'
```

Comment:
How are the runners tagged? I'm wondering if we submit this PR will we not have any FPGAs with this label applied?

Reply:
They are tagged offline. I've configured two runners with this label.